BCC/src/main/java/BNU/logic/LoginController.java

```diff
@@ -68,7 +68,10 @@ public void actionPerformed(ActionEvent e) {
 		} else if (e.getActionCommand() == "login:Authorize") {
 			System.out.println("login:Authorize Button Pressed");
 			WindowBuilder.loadPage(new MainController());
+		} else if(e.getActionCommand() == "login:AuthorizeAdmin") {
```

Comment:
the login controller isn't going to be taking care of this, The mainView will only display a button that grants access to the admin controller page if they are an admin